packages/kbn-recently-accessed/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/recently-accessed
+
+Empty package generated by @kbn/generate
```

Comment:
```suggestion
The `RecentlyAccessedService` uses browser local storage to manage records of recently accessed saved objects. This can be used to make recent items easier for users to find in listing UIs.
```